css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,49 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "DM sans", sans-serif;
+  color: #e6e6e6;
+}
+
+.container {
+  width: 1055px;
+  margin: 0 auto;
+}
+
+.heading {
+  text-align: center;
+  color: black;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  width: 190px;
+  margin: 0 10px;
+}
+
```

Comment:
`column` what? You shouldn't set column widths in pixel values, you should use percentages that match their intended size.